### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by dhirschfeld**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by dhirschfeld** [comment]
@conda-forge-admin, please rerender

**Comment by dhirschfeld** [comment]
***xref:***
* https://github.com/arviz-devs/arviz/blame/v0.12.1/requirements.txt
* https://github.com/arviz-devs/arviz/pull/1948

**Comment by dhirschfeld** [comment]
Ping @conda-forge/arviz! It would be great to get this merged as it's preventing me from installing `pymc`.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ source:
   sha256: 57d80eacc51909f18e6ab63c96a6b02227c3b077c5ffa406d5f4dabe03b8f019
 build:
   noarch: python
-  number: 0
+  number: 1
```
Comment (canyon289) [inline recipe/meta.yaml]:
```suggestion
  number: 0
```

**Comment by dhirschfeld-shell** [comment]
Hi @canyon289 :wave:,
When you publish a new package with the *same version* you need to bump the build number:

image

I'm not entirely sure that's the reason that the package didn't upload:
https://anaconda.org/conda-forge/arviz/files

...but even if it did upload, mamba/conda wouldn't necessarily choose to install it as (IIUC) the resolution between two packages with the same version and build number would resort to alphabetical which would depend on the random build string.

**Comment by dhirschfeld** [comment]
Hi @canyon289 :wave:,
When you publish a new package with the *same version* you need to bump the build number:

image

I'm not entirely sure that's the reason that the package didn't upload:
https://anaconda.org/conda-forge/arviz/files

...but even if it did upload, mamba/conda wouldn't necessarily choose to install it as (IIUC) the resolution between two packages with the same version and build number would resort to alphabetical which would depend on the random build string.

**Pull request merged**